02-electric/electric-section-one/electric-section-one.css

```diff
@@ -17,71 +31,296 @@
 
 .electric-section-one-title {
   text-align: center;
-  font-size: var(--title-size-mobile);
+  font-size: var(--title-font-size-mobile);
+  font-weight: var(--title-font-weight-mobile);
 }
 
-.electric-section-one-subtitle {
-  text-align: center;
-  font-size: var(--title-size-mobile);
+@media (min-width: 500px) {
+  .electric-section-one-title {
+    text-align: center;
```

Comment:
this is not neccesary because its defined in mobile first and never changes

Reply:
done